### DISCUSSION TITLE
OpenSeaMap support

### DISCUSSION THREAD

**Discussion by dataselfservice** [grafana/grafana#47812] [Dashboards, Panels & Visualizations]
Hi, 

any plan to include OpenSeaMap (https://www.openseamap.org) e.g. in the Geomap plugin?

It is a "marks only" map type (https://wiki.openstreetmap.org/wiki/Tile_servers), therefore it would require the possibility to overlap 2 layers.

We might invest in adding this, any indication what is best way forward to do it?

Regards,
DSS team

**Comment by pbegg** [comment]
Also very intersted in this

**Comment by aloodigital** [comment]
@dataselfservice @pbegg I made a PR for this. Check it out at [https://github.com/grafana/grafana/pull/49528](url)

**Comment by nmarrs** [comment]
Hi @dataselfservice, Geomap now includes the OpenSea layer. It was included as part of [this PR](https://github.com/grafana/grafana/pull/50053)

It is included in v9.1+

**Comment by dataselfservice** [comment]
Hi,
great and thank you.
DataSelfService team

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
